LINK/usr/bin/appliance_console_cli

```diff
@@ -1,18 +1,15 @@
 #!/usr/bin/env ruby
 
 require "pathname"
-# NOTE: in dev, running in cfme/system/LINK/bin/appliance_console_cli
-ROOT ||= [
-  "/var/www/miq",
-  File.expand_path(File.join(File.dirname(__FILE__), "../../.."))
-].detect { |f| File.exist?(f) }
-RAILS_ROOT ||= Pathname.new("#{ROOT}/vmdb")
 
-ENV['BUNDLE_GEMFILE'] ||= "#{ROOT}/vmdb/Gemfile"
+# Handle development environment by setting up Gemfile and manageiq location
+ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../../../manageiq/Gemfile', __dir__)
```

Comment:
nope. take it all back. `appliance_console_cli` is no longer in the manageiq repo.

ok, developers can not call `appliance_console_cli`.
Darn, this means we will have 2 ways to do all these tasks, one for devs and one for production. this is very concerning to me, but unrelated to this PR.

:+1: :shipit: